app/trans/http/handler.go

```diff
@@ -0,0 +1,31 @@
+package http
+
+import (
+	"net/http"
+
+	"github.com/go-devs-ua/octagon/app/ent"
+)
+
+type User = ent.User
+
+// ApiHandler is User HTTP handler
+type ApiHandler struct {
+	logic UserLogic
+}
+
+// NewApiHandler will return *ApiHandler
+// accepting UseCases interface
+func NewApiHandler(logic UserLogic) *ApiHandler {
+	return &ApiHandler{
+		logic: logic,
+	}
+}
+
+// HandleUser will take care of our handsome User's endpoint
+func (ah *ApiHandler) HandleUser() http.Handler {
```

Comment:
After all, it can look like "func (u *UserHandler) CreateUser() http.Handler {"

Reply:
Not exactly... I will try to make myself clear